src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -34,15 +34,14 @@ export class TaskListBuilder {
       OweMoneyTask.isCompleted(draft)))
 
     if (draft.requireHowMuchPaid()) {
```

Comment:
This reads wired. When `requireHowMuchPaid()` condition evaluates to true we add `'How much money do you believe you owe?'` to the task list. In turn when `requireHowMuchOwed()` evaluates to true we add `'How much have you paid the claimant?'` to the task list. My first impression is that `much paid` and `much owed` have swapped place.

Reply:
Yes thats right, it is a bit confusing, which is why originally they were in that order, then having had a look at the updated prototype they needed to be swapped

Reply:
Yeah, this will keep confusing people each time anybody opens that bit of code. We should come up with better naming.